src/cpp/session/modules/quarto/SessionQuarto.cpp

```diff
@@ -616,6 +616,40 @@ SEXP rs_quartoFileResources(SEXP targetSEXP)
    return r::sexp::create(resources, &protect);
 }
 
+SEXP rs_quartoFileProject(SEXP basenameSEXP, SEXP dirnameSEXP)
```

Comment:
I think this implementation needs to be guarded by `QUARTO_ENABLED` as some platforms don't support Quarto.

Reply:
Made those changes in https://github.com/rstudio/rstudio/commit/39de73fd61812d19e1f070188eea14ddc932d84d.